src/app/modules/dashboard/pages/settings/settings.page.html

```diff
@@ -9,7 +9,7 @@
 
 <div class="page__wrapper">
   <div class="row">
-    <div class="col-4">
+    <div class="col-lg-6">
```

Comment:
Revert this, or maybe col-lg-4 could work as well.

Reply:
changed to `col-lg-4`
This is how it looks now
https://user-images.githubusercontent.com/15015936/113489388-3e05cb00-94c4-11eb-99bc-da0be94ab4c2.mov

but I still think `col-lg-6` looks better :) ?
cheers